README.md

```diff
@@ -1 +1,8 @@
-# Code-review-practice
\ No newline at end of file
+# Code-review-practice
+
+This repo contians copied code from another repo for the purpose of code review practice. 
+The original code was written while applying to Microverse. 
+
+
+## Contributors
+It was written by me and "Edgar Rincon Serrano"
```

Comment:
[Optional]
It's better to add links for contributors' accounts to navigate through their accounts if needed easily.
You can use [this link](https://github.com/microverseinc/readme-template) as a reference.